helpers/crearMovimientoDetalle.go

```diff
@@ -116,23 +175,20 @@ func CrearMovimientoDetalle(
 			err := "Tanto el saldo como el valor tienen un valor diferente de 0, no se puede añadir el movimiento detalle"
 			panic(errorctrl.Error("crearMovimientoDetalle - saldo != 0 && valor != 0", err, "400"))
 		}
-	} else {
-		if saldo != valor {
-			err := "El saldo y el valor de publicación debe ser el mismo"
-			panic(errorctrl.Error("crearMovimientoDetalle - saldo != valor", err, "400"))
-		}
 	}
 
 	detalleCuenPre := cuentaMovimientoDetalle.Cuen_Pre
 
+	logs.Debug("DETALLE CUENTA PRESUPUESTAL: ", detalleCuenPre)
```

Comment:
Dejar comentada o eliminar
```suggestion
	// logs.Debug("DETALLE CUENTA PRESUPUESTAL: ", detalleCuenPre)
```